diff --git a/bin/apply_osc_to_db.sh b/bin/apply_osc_to_db.sh
index c1a56960..06ffe3c6 100644
--- a/bin/apply_osc_to_db.sh
+++ b/bin/apply_osc_to_db.sh
@@ -114,9 +114,6 @@ update_state()
   DATA_VERSION=${TIMESTAMP_LINE:10}
 };

-
-echo >>$DB_DIR/apply_osc_to_db.log
-
 mkdir -p $DB_DIR/augmented_diffs/
 DIFF_COUNT=0

@@ -131,20 +128,20 @@ while [[ true ]]; do
     START=`cat $DB_DIR/replicate_id`
   }; fi

-  echo "`date -u '+%F %T'`: updating from $START" >>$DB_DIR/apply_osc_to_db.log
+  echo "updating from $START"

   TEMP_DIR=`mktemp -d /tmp/osm-3s_update_XXXXXX`
   collect_minute_diffs $TEMP_DIR

   if [[ $TARGET -gt $START ]]; then
   {
-    echo "`date -u '+%F %T'`: updating to $TARGET" >>$DB_DIR/apply_osc_to_db.log
+    echo "updating to $TARGET"

     update_state
     apply_minute_diffs $TEMP_DIR
     echo "$TARGET" >$DB_DIR/replicate_id

-    echo "`date -u '+%F %T'`: update complete" $TARGET >>$DB_DIR/apply_osc_to_db.log
+    echo "update complete" $TARGET
   };
   else
   {
diff --git a/bin/fetch_osc.sh b/bin/fetch_osc.sh
index d81156d2..87970862 100644
--- a/bin/fetch_osc.sh
+++ b/bin/fetch_osc.sh
@@ -76,7 +76,7 @@ retry_fetch_file()

 file_panic()
 {
-  echo "fetch_osc()@"`date -u "+%F %T"`": upstream_delay $REPLICATE_ID" >>$LOCAL_DIR/fetch_osc.log
+  echo "upstream_delay $REPLICATE_ID"
   REPLICATE_ID=$(($REPLICATE_ID - 1))

   printf -v TDIGIT3 %03u $(($REPLICATE_ID % 1000))
@@ -97,9 +97,9 @@ file_panic()
   RES_GZIP=`diff -q "$LOCAL_PATH/$TDIGIT3.osc.gz" "$LOCAL_PATH/$TDIGIT3.new.osc.gz"`
   RES_TEXT=`diff -q "$LOCAL_PATH/$TDIGIT3.state.txt" "$LOCAL_PATH/$TDIGIT3.new.state.txt"`
   if [[ -n $RES_GZIP || -n $RES_TEXT ]]; then
-    echo "fetch_osc()@"`date -u "+%F %T"`": file_panic $REPLICATE_ID" >>$LOCAL_DIR/fetch_osc.log
-    echo "fetch_osc()@"`date -u "+%F %T"`": $RES_GZIP" >>$LOCAL_DIR/fetch_osc.log
-    echo "fetch_osc()@"`date -u "+%F %T"`": $RES_TEXT" >>$LOCAL_DIR/fetch_osc.log
+    echo "file_panic $REPLICATE_ID"
+    echo "$RES_GZIP"
+    echo "$RES_TEXT"
     exit 1
   fi

@@ -137,7 +137,7 @@ do
 {
   REPLICATE_ID=$(($REPLICATE_ID + 1))
   fetch_minute_diff
-  echo "fetch_osc()@"`date -u "+%F %T"`": new_replicate_diff $REPLICATE_ID $TIMESTAMP" >>$LOCAL_DIR/fetch_osc.log
+  echo "new_replicate_diff $REPLICATE_ID $TIMESTAMP"
   sleep 1
 };
 done
diff --git a/bin/fetch_osc_and_apply.sh b/bin/fetch_osc_and_apply.sh
index d51d9f1b..e623c58d 100644
--- a/bin/fetch_osc_and_apply.sh
+++ b/bin/fetch_osc_and_apply.sh
@@ -121,9 +121,6 @@ update_state()
   DATA_VERSION=${TIMESTAMP_LINE:10}
 };

-
-echo >>$DB_DIR/fetch_osc_and_apply.log
-
 mkdir -p $DB_DIR/augmented_diffs/
 DIFF_COUNT=0

@@ -134,7 +131,7 @@ START=`cat $DB_DIR/replicate_id`

 while [[ true ]]; do
 {
-  echo "`date -u '+%F %T'`: updating from $START" >>$DB_DIR/fetch_osc_and_apply.log
+  echo "updating from $START"

   TEMP_SOURCE_DIR=`mktemp -d /tmp/osm-3s_update_XXXXXX`
   TEMP_TARGET_DIR=`mktemp -d /tmp/osm-3s_update_XXXXXX`
@@ -142,13 +139,13 @@ while [[ true ]]; do

   if [[ $TARGET -gt $START ]]; then
   {
-    echo "`date -u '+%F %T'`: updating to $TARGET" >>$DB_DIR/fetch_osc_and_apply.log
+    echo "updating to $TARGET"

     update_state
     apply_minute_diffs $TEMP_TARGET_DIR
     echo "$TARGET" >$DB_DIR/replicate_id

-    echo "`date -u '+%F %T'`: update complete" $TARGET >>$DB_DIR/fetch_osc_and_apply.log
+    echo "update complete" $TARGET
   };
   else
   {
